geohosting/www/portal/support_tickets.html

```diff
@@ -12,6 +12,56 @@
 <script src="/assets/geohosting/js/util.js"></script> 
 <style>
     
+.card {

```

Comment:
@ketankartoza can you also not use a modal to edit the ticket rather use the existing form for creating a ticket . So when user click on ticket it should repeat the action similar to when user clicks new ticket ...then on that form opened show the details of the ticket being edited